waspc/cli/Command/Compile.hs

```diff
@@ -14,12 +19,23 @@ import qualified StrongPath             as SP
 import qualified Util.IO
 
 
+compile :: Command ()
```

Comment:
It is helpful for me as dev to be able to just compile sometimes without other stuff, so I added this command as we discussed.